packages/graphql/src/schemaTypes/objectTypes/gql-Project.ts

```diff
@@ -10,7 +10,7 @@ export interface ProjectShape {
   projectRoot: string
 }
 
-const GeneratedSpec = objectType({
+export const GeneratedSpec = objectType({
```

Comment:
For this I think we should keep it to one objectType per file. It would be easier to find `gql-GeneratedSpec` then remember what it's paired with